.github/workflows/snek.yml

```diff
@@ -0,0 +1,21 @@
+name: Snek
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Could we use debian unstable instead? The dependencies are all available there.

Reply:
Not really... 

<img width="681" alt="Screen Shot 2019-11-19 at 18 09 35" src="https://user-images.githubusercontent.com/1095/69186690-ca923200-0af7-11ea-84ec-b99c0df152e8.png">

Reply:
Looks like we can create a custom action using a docker container and use it to build artifacts, will investigate. Still learning about what is possible on this.